opensrp-reveal/src/main/java/org/smartregister/reveal/view/ListTasksActivity.java

```diff
@@ -401,9 +406,15 @@ public void onClick(View v) {
             drawerView.openDrawerLayout();
         } else if (v.getId() == R.id.progressIndicatorsGroupView) {
             openIndicatorsCardView();
+        } else if (v.getId() == R.id.filter_tasks_fab) {
+            openFilterTaskActivity();
         }
     }
 
+    private void openFilterTaskActivity() {
+        startActivityForResult(new Intent(this, FilterTasksActivity.class), RequestCode.REQUEST_CODE_FILTER_TASKS);
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Unnecessary use of fully qualified name 'RequestCode.REQUEST_CODE_FILTER_TASKS' due to existing static import 'org.smartregister.reveal.util.Constants.RequestCode.REQUEST_CODE_FILTER_TASKS'](https://app.codacy.com/manual/OpenSRP/opensrp-client-reveal/pullRequest?prid=4684568)